bin/pipotd

```diff
@@ -13,6 +13,7 @@ function usage() {
     echo -e "\t\t--help\tThis help\n"
 }
 
+export PYTHONPATH=/usr/src/pipot/server/
```

Comment:
a) This is hardcoded (not good)
b) Is this really necessary? If launched properly it shouldn't be necessary I think.

Reply:
change it by using $DIR

I think it is reasonable for it to have import error. As the pwd here is server/bin, while config_parser and  others are under server/.